src/Models/HydrostaticFreeSurfaceModels/update_hydrostatic_free_surface_model_state.jl

```diff
@@ -26,26 +26,155 @@ hydrostatic pressure) to the current model state. If `callbacks` are provided (i
 they are called in the end.
 """
 update_state!(model::HydrostaticFreeSurfaceModel, callbacks=[]; compute_tendencies = true) =
-         update_state!(model, model.grid, callbacks; compute_tendencies)
+    update_state!(model, model.grid, callbacks; compute_tendencies)
+
+operation_corner_points = "default" # Choose operation_corner_points to be "default", "average", "CCW", or "CW".
+
+function fill_paired_halo_regions!(fields, signed=true)
```

Comment:
hm... do we need this method to fill the metric in the grid?

Reply:
OK, I see we do need it. Right?

Reply:
Yes, in line 250 of src/MultiRegion/multi_region_cubed_sphere_grid.jl, we are using this function to fill the halos of the grid metrics of the conformal cubed sphere with signed = false.